network-api/networkapi/wagtailcustomization/views/snippet_chooser.py

```diff
@@ -0,0 +1,57 @@
+from wagtail.admin.forms.choosers import (
+    BaseFilterForm,
+    CollectionFilterMixin,
+    SearchFilterMixin,
+)
+from wagtail.admin.views.generic.chooser import (
+    ChooseResultsViewMixin,
+    ChooseViewMixin,
+    CreationFormMixin,
+)
+from wagtail.admin.viewsets.chooser import ChooserViewSet
+from wagtail.models import CollectionMember, Locale, TranslatableMixin
+from wagtail.search.index import class_is_indexed
+from wagtail.snippets.views.chooser import BaseSnippetChooseView
+
+
+class NoLocaleSnippetChooseView(ChooseViewMixin, CreationFormMixin, BaseSnippetChooseView):
```

Comment:
I think `DefaultLocaleSnippetChooseView` would be a better name, as we are only getting the default locale snippets (instead of no locale)

Reply:
That has been renamed now.